notification_center.py

```diff
@@ -30,7 +30,7 @@
 	if not weechat.config_is_set_plugin(key):
 		weechat.config_set_plugin(key, val)
 
-weechat.hook_print('', 'irc_privmsg', '', 1, 'notify', '')
+weechat.hook_print('', ', '', 1, 'notify', '')
```

Comment:
You have a typo in there. Make sure you test this change in Weechat, as I no longer use Weechat, so I can't test.

Reply:
Will fix. I have this running in wee-chat now, so I'm not sure how I missed that when I pushed. It's definitely correct locally now.